src/infrastructure/group-store/group-store.test.ts

```diff
@@ -0,0 +1,164 @@
+import "reflect-metadata";
+import { DependencyContainer } from "tsyringe";
+import { GroupStore } from "../../topics/group";
+import { getTestGroups, exampleData } from "../../topics/group/test-groups";
+import { getMemoryContainer, getTestLocalContainer } from "../container";
+
+const getTestMemoryContainer = () => {
+  const container = getMemoryContainer();
+  container.clearInstances();
+  return container;
+};
+
+const getGroupStore = (container: DependencyContainer): GroupStore => {
+  return container.resolve("GroupStore");
+};
+
+describe("test groups stores", () => {
+  const testCases = [[getTestMemoryContainer], [getTestLocalContainer]];
+
+  it.each(testCases)(
+    "Should generate a group and retrieve from store",
+    async (getContainer) => {
+      const container = getContainer();
```

Comment:
Copy pasta code, i can't use the real use case of what we are testing in those tests...

Reply:
Fixed !